Routing/RouteRegistry.cs

```diff
@@ -1,37 +1,187 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
+using Routing.Parsing;
+using Routing.SegmentVariant;
 
 namespace Routing
 {
     public class RouteRegistry<TResponse, TRequest> : IRouteRegistry<TResponse, TRequest>
```

Comment:
`TRequest` and `TResponse` should be swapped.

Reply:
https://github.com/messerli-informatik-ag/routing/pull/6/commits/2498e174f580c386d1be3efa81eb09a9db22a977